publicationiq.json

```
@@ -5001,6 +5001,151 @@
           }
         ]
       }
+    },
+    "/pf/v1/pfaccount/{profile}/alphabrowser/menu/{localeid}": {

```

```suggestion
    "/v1/pf/pfaccount/{profile}/alphabrowser/menu/{localeid}": {

```
to put it in alignment with the other V1 endpoint on PubIQ. I think it would be more confusing for end users to have it swapped between endpoints.